native/cocos/scene/Camera.cpp

```diff
@@ -110,7 +111,6 @@ void Camera::destroy() {
         _window = nullptr;
     }
     _name.clear();
-    _geometryRenderer->destroy();
```

Comment:
Is it better to keep this line and  set _geometryRenderer to nullptr?
```c++
Camera::~Camera() {
    CC_SAFE_DESTROY_NULL(_geometryRenderer); // if _geometryRenderer is nullptr, then do nothing.
    _frustum->release();
}

void Camera::destroy() {
    if (_window) {
        _window->detachCamera(this);
        _window = nullptr;
    }
    _name.clear();

    CC_SAFE_DESTROY_NULL(_geometryRenderer); // If _geometryRenderer is not nullptr, invoke destroy method and set it to nullptr.
```

Reply:
1